src/main/java/com/celements/metatag/MetaTagApi.java

```diff
@@ -0,0 +1,94 @@
+package com.celements.metatag;
+
+import java.util.Map;
+
+import javax.validation.constraints.NotNull;
+
+import com.celements.metatag.enums.ECharset;
+import com.celements.metatag.enums.EHttpEquiv;
+import com.celements.metatag.enums.ENameNonStandard;
+import com.celements.metatag.enums.ENameStandard;
+import com.celements.metatag.enums.EReferrer;
+import com.celements.metatag.enums.ERobot;
+import com.celements.metatag.enums.EViewport;
+import com.celements.metatag.enums.opengraph.EOpenGraph;
+import com.celements.metatag.enums.twitter.ETwitter;
+import com.celements.metatag.enums.twitter.ETwitterCardType;
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableMap;
+
+public final class MetaTagApi {
```

Comment:
Like you said, I mostly understood the XXXApi class to be a wrapper which prevents rights escalation for an XXX class. The latter doesn't exist in this case though.

I know that the "Api"-naming stems from XWiki, but I would actually prefer a discrete annotation to generally mark java classes which are passed to scripting languages. E.g. we don't put "Singleton" into the naming since we can note it @Singleton.